Header header
int16 left
int16 right

